tests/kernel/nvmftests.pm

```diff
@@ -0,0 +1,87 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 Michael Moese <mmoese@suse.de>
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: new test that runs NVMe over Fabrics testsuite
+# Maintainer: Michael Moese <mmoese@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use utils 'zypper_call';
+use serial_terminal 'select_virtio_console';
+
+sub run {
+    select_virtio_console();
+
+    zypper_call('ar -f -G ' . get_required_var('BENCHMARK_REPO'));
+    zypper_call('ar -f -G ' . get_required_var('DEVEL_LANG_PYTHON_REPO'));
```

Comment:
Why would it be necessary to add a custom repo? We should rely at best only on packages in the main repo

Reply:
The nvmftests package is not in the main repo and will not be. 
The devel:languages:python is required to meet the requirements of nvmftests.